pyvista/plotting/camera.py

```diff
@@ -848,7 +848,7 @@ def tight(self, padding=0.0, adjust_render_window=True, view='xy', negative=Fals
         >>> pl.show()
 
         """
-        # inspired by vedo resetCamera. Thanks @marcomusy!
+        # inspired by vedo resetCamera. Thanks @marcomusy
```

Comment:
How about
```suggestion
        # inspired by vedo resetCamera; thanks @marcomusy
```
?

Reply:
From ChatGPT:
```
Using a semicolon in this context is not the most appropriate choice. It appears that you are trying to use the semicolon as a separator between two phrases: "inspired by vedo resetCamera" and "thanks @marcomusy". In this case, a semicolon is not necessary and can be replaced with a comma or a period.

Here are a few alternatives:

Inspired by vedo resetCamera, thanks @marcomusy.
Inspired by vedo resetCamera. Thanks @marcomusy.
A semicolon is typically used to separate two closely related independent clauses. For example:

I went to the store; they were out of milk.
```

This agrees with https://academicguides.waldenu.edu/writingcenter/punctuation/semicolons

The ideal I think is:

```suggestion
        # Inspired by vedo resetCamera. Thanks @marcomusy.
```